WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -16045,14 +16069,6 @@ e_armenia = {
 		zoroastrian_group = 200 # Crusade target weight
 		
 		# Creation/usurpation trigger
-		allow = {
-			hidden_tooltip = {
-				OR = {
-					ai = no
-					religion_group = christian
-				}
-			}
-		}
```

Comment:
Yeah, you have removed the entire # Creation/usurpation trigger. What are you trying to say?